tensorflow/python/framework/ops.py

```diff
@@ -253,9 +254,24 @@ def disable_tensor_equality():
   Tensor._USE_EQUALITY = False  # pylint: disable=protected-access
 
 
+DataType = TypeVar("DataType",
+                   dtypes.Float16, dtypes.Float32, dtypes.Float64,
+                   dtypes.BFloat16, dtypes.Complex64, dtypes.Complex128,
+                   dtypes.Int8, dtypes.UInt8, dtypes.UInt16,
+                   dtypes.UInt32, dtypes.UInt64, dtypes.Int16,
+                   dtypes.Int32, dtypes.Int64, dtypes.Bool,
+                   dtypes.String, dtypes.QInt8, dtypes.QUInt8,
+                   dtypes.QInt16, dtypes.QUInt16, dtypes.QInt32,
+                   dtypes.Resource, dtypes.Variant)
+
+if sys.version_info[0] >= 3:
```

Comment:
Please include a TODO to remove this once compatibility with Python 2 is dropped.